engine/src/main/java/org/terasology/rendering/nui/widgets/UIDropdownScrollable.java

```diff
@@ -312,44 +312,6 @@ public void setOpenedReverse(boolean selectionSet) {
         }
     }
 
-    public void changeHighlighted(boolean increase) {
```

Comment:
Something in this removal might have removed the intended yellow highlight box around a UI widget that has been selected by tabbing to it?

Reply:
IIRC `highlighted` is actually for which item in the dropdown is currently selected (via tabbing or otherwise) because that gets a box around it as well. I think the culprit is in the parent class - `UIDropdown` - where it determines if the widget is in active mode or not.